src/sync/sync-utils.ts

```diff
@@ -22,7 +22,8 @@ export const findOrStartSync = async (
 	await subscription.update({
 		syncStatus: SyncStatus.PENDING,
 		numberOfSyncedRepos: 0,
-		syncWarning: null
+		syncWarning: null,
+		failedCode: null
```

Comment:
Do we have `failedCode` in `Subscription`?

Reply:
destroyed